### PR TITLE
S3 Region extraced to ENV

### PR DESCRIPTION
**Pull request by brybacki**
**What this PR does / why we need it**:
This is a follow up to https://github.com/kubevirt/kubevirt-velero-plugin/pull/25. One Variable was missing the S3 Region.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/framework/framework.go [hunk 1]
```diff
@@ -45,6 +49,17 @@ func getBackupNamespaceFromEnv() string {
 	return backupNamespace
 }
 
+func getRegionFromEnv() string {
+	region := os.Getenv(regionEnv)
+	if region == "" {
+		fmt.Fprintf(os.Stderr, "defaulting to minio ns\n")
```
Comment (arnongilboa) [inline tests/framework/framework.go:55]: Consider using a log wrapper func to replace all `fmt.Fprintf(os.Stderr,...)` calls

Comment (brybacki) [reply tests/framework/framework.go:55]: Good Idea, I'll do it in next PR, I really want to test the velero plugin today on some other environments, and this PR will help to make it happed.

**Comment by brybacki** [comment]
the failure here does not look good, the DV has correct annotation `"cdi.kubevirt.io/storage.prePopulated": "test-vm-dv"` but anyway DV is in phase `"phase": "ImportScheduled"`

This might happen if CDI has a bug, or if we wrongly  interpret ImportScheduled. Will create an issue for that.

**Comment by brybacki** [comment]
card already exists, looks like I need to make a priority to check this
https://github.com/kubevirt/kubevirt-velero-plugin/issues/14

**Pull request merged**